processor/processor.go

```diff
@@ -815,6 +917,16 @@ func (proc *HandleT) processJobsForDest(jobList []*jobsdb.JobT, parsedEventList
 		proc.batchRouterDB.Store(batchDestJobs)
 	}
 
+	var procErrorJobs []*jobsdb.JobT
+	for _, jobs := range procErrorJobsByDestID {
+		procErrorJobs = append(procErrorJobs, jobs...)
+	}
+	if len(procErrorJobs) > 0 {
+		logger.Debug("[Processor] Total jobs written to proc_error: ", len(procErrorJobsByDestID))
```

Comment:
Mark this as Info/Error instead of debug?

Reply:
done